fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1679,6 +1679,13 @@ public class Config extends ConfigBase {
     @ConfField(mutable = false, masterOnly = false)
     public static long max_hive_partition_cache_num = 100000;
 
+    /**
+     * Max cache num hive table to partition name list.
+     * Decrease this value if FE's memory is small
+     */
+    @ConfField(mutable = false, masterOnly = false)
+    public static long max_hive_table_partition_value_num = 100;
```

Comment:
```suggestion
    public static long max_hive_table_catch_num = 1000;
```

Reply:
Please refer to this https://github.com/apache/doris/pull/19246
to add config's document